src/gui/transferlistfilterswidget.h

```diff
@@ -129,7 +129,7 @@ class TrackerFiltersList final : public BaseFilterWidget
     void removeItem(const QString &trackerURL, const BitTorrent::TorrentID &id);
     void changeTrackerless(bool trackerless, const BitTorrent::TorrentID &id);
     void setDownloadTrackerFavicon(bool value);
-    void handleTrackerEntriesUpdated(const QHash<BitTorrent::Torrent *, QHash<QString, BitTorrent::TrackerEntryUpdateInfo>> &updateInfos);
+    void handleTrackerEntriesUpdated(const QHash<BitTorrent::Torrent *, QSet<QString>> &updateInfos);
```

Comment:
> I suppose passing QVector (or QStringList) instead of QSet would be better, just avoid adding duplicates when constructing it. Would that be feasible?

Do you suggest checking for duplicates in QStringList?
Otherwise (if initially create QSet) there is no point of transforming it in QStringList in this case, IMO.